### DISCUSSION TITLE
Multiple Categories for add post

### DISCUSSION THREAD

**Discussion by Pizza1** [wp-graphql/wp-graphql#3254] [General] [answered]
Hi,

How to add multiple categories when adding a post?

![image](https://github.com/user-attachments/assets/d0b0327d-a705-4d1b-ac15-a577475c8fab)

**Comment by justlevine** [comment, marked as answer]
Not by my desk to check, but I think you can pass an array to `nodes` i.e. `nodes[ {slug: "cat1"}, {slug:"cat2"} ]`, but you can open up the Documentation Explorer in the right-side panel of the IDR to inspect the argument to confirm.
